### PR TITLE
Iteration page object model and few test

### PR DESCRIPTION
**Pull request by naina-verma**
Iteration page object model and few test
- Verification of elements are in place 
- Verification set the title and description
- Verify  help box / dialog title is visible

### CODE REVIEW

**Comment by naina-verma** [comment]
[test]

File: src/tests/work-item/work-item-list/page-objects/work-item-list.page.js [hunk 1]
```diff
@@ -276,13 +276,37 @@ class WorkItemListPage {
   get filterDropdown () {
     return  element(by.id("wi_filter_dropdown"));
   }
-  filterDropdownId () {
```
Comment (ldimaggi) [inline src/tests/work-item/work-item-list/page-objects/work-item-list.page.js]: So - this is not needed? (Was it ever used?) Thx!

File: src/tests/work-item/work-item-list/page-objects/work-item-list.page.js [hunk 2]
```diff
@@ -432,8 +450,42 @@ class WorkItemListPage {
...
+    return this.iterationTitle.sendKeys(newTitleString);
+  }
+  get iterationDescription  (){
+    return element(by.id("iteration-description")).click();
+  }
+  setIterationDescription  (newString,append){
+     if (!append) { this.iterationDescription.clear(newString) };
+    return this.iterationDescription.sendKeys(newString);
+  }
+  clickCreateIteration  (){
```
Comment (ldimaggi) [inline src/tests/work-item/work-item-list/page-objects/work-item-list.page.js]: We should have separate functions to define the create and cancel buttons, and to click the buttons. Having the 2 buttons will keep the page object model elements consistent. Thx!

File: src/tests/work-item/work-item-list/page-objects/work-item-list.page.js (hunk 1 above)
Comment (naina-verma) [inline src/tests/work-item/work-item-list/page-objects/work-item-list.page.js]: I discovered this duplicated function during this PR , it looks like never used so removed it :)

File: src/tests/work-item/work-item-list/page-objects/work-item-list.page.js (hunk 2 above)
Comment (naina-verma) [inline src/tests/work-item/work-item-list/page-objects/work-item-list.page.js]: I  did not make separate functions to define button and directly used them, bcuz I  couldn't think of any other use case rather then clicking on it for now :). And yes we can use the general method for checking the presence of the elements. 

I will add general definition in different methods.

**Comment by naina-verma** [comment]
[test]

**Comment by naina-verma** [comment]
[test]

**Comment by naina-verma** [comment]
[test]

**Comment by naina-verma** [comment]
[test]

**Pull request merged**